src/argilla_sdk/records/_io/_json.py

```diff
@@ -40,7 +36,7 @@ def to_json(records: List["Record"], path: Union[Path, str]) -> Path:
             path = Path(path)
         if path.exists():
             raise FileExistsError(f"File {path} already exists.")
-        record_dicts = [GenericIO._record_to_dict(record) for record in records]
+        record_dicts = [Record.to_dict(record) for record in records]
```

Comment:
Does this make sense?
```suggestion
        record_dicts = GenericIO.to_list(records)
```